src/Chapter1.hs

```diff
@@ -508,7 +508,9 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if abs x < abs y
+                    then x
+                    else y
```

Comment:
Perfect 👌🏼 

Usually, we try to not make the indentation to depend on the previous lines, as you see, if you decide to rename the function, you will need to realign both `then` and `else` lines as well.

I would personally write like this:

```haskell
closestToZero x y = 
    if abs x < abs y
    then x
    else y
```

But this is just a note! This doesn't affect the correctness of your solution at all! 🌟